.gitignore

```diff
@@ -14,3 +14,4 @@ coverage/
 # Dependency directory
 node_modules/
 bower_components/
+ts/typings/
```

Comment:
For sure flatten it (remove `ts/` directory). It's my mistake that it was here :|